### DISCUSSION TITLE
Running DIA-NN in HPC

### DISCUSSION THREAD

**Discussion by akila-wijerathna-yapa** [vdemichev/DiaNN#867] [Q&A] [answered]
Hi Vadim,
I SCIEX ZenoTOF 7600 system data and I would like to do my data analysis in DIA-NN.

However, now I have to move to my data analysis to a HPC server as my laptop can not handle big data (I have about 120 sample).

When I was working on GUI/windows DIA-NN, I follow a protocol copying “SCIEX and Clearcore” in Proteowizard 64bit folder to the DIA-NN root folder. However, since I have to move to Linux based server now, how can I run DIA-NN for SCIEX files?

Also what are settings I should changed in command line to get defaults settings of GUI software?

I would be really grateful if you could share me helpful guideline/protocol s if possible (I'm new to Linux and HPC).

**Comment by vdemichev** [comment, marked as answer]
Hi Akila,

On Linux it's worth trying things under Wine, but I don't remember if it works with Sciex libs.
You can always do conversion to dia or mzML under Windows and then analyse those under Linux natively.

'Also what are settings I should changed in command line to get defaults settings of GUI software?' - GUI prints the exact command it passes to diann.exe, so need to just copy that.

Best,
Vadim

**Reply by akila-wijerathna-yapa** [reply]
Thanks a heap Vadim.
